x-pack/docs/en/security/authorization/alias-privileges.asciidoc

```diff
@@ -64,100 +63,44 @@ GET .ds-my-data-stream-2099.03.09-000003/_doc/2
 // TEST[s/.ds-my-data-stream-2099.03.09-000003/my-index/]
 
 [[index-alias-privileges]]
-==== Index alias privileges
+==== Alias privileges
+
+Use <<privileges-list-indices,index privileges>> to control access to an
+<<alias,alias>>. Privileges on an index or data stream do not grant privileges
+on its aliases.
 
-An index alias points to one or more indices,
-holds metadata and potentially a filter. The {es} {security-features} treat
-aliases and indices
-the same. Privileges for indices actions are granted on specific indices or
-aliases. In order for an indices action to be authorized, the user that executes
-it needs to have permissions for that action on all the specific indices or
-aliases that the request relates to.
+IMPORTANT: Don't use <<filter-alias,filtered aliases>> in place of
+<<document-level-security,document level security>>. {es} doesn't always apply
+alias filters.
 
-Let's look at an example. Assuming we have an index called `2015`, an alias that
-points to it called `current_year`, and a user with the following role:
+For example, the `current_year` alias points only to the `2015` index. A user is
+granted the `read` privilege to the `2015` index.
 
 [source,js]
---------------------------------------------------
+----
 {
   "names" : [ "2015" ],
   "privileges" : [ "read" ]
 }
---------------------------------------------------
+----
 // NOTCONSOLE
 
-The user attempts to retrieve a document from `current_year`:
+When the user attempts to retrieve a document from the `current_year` alias,
+{es} rejects the request.
 
 [source,console]
--------------------------------------------------------------------------------
-GET /current_year/_doc/1
--------------------------------------------------------------------------------
+----
+GET current_year/_doc/1
+----
 // TEST[s/^/PUT 2015\n{"aliases": {"current_year": {}}}\nPUT 2015\/_doc\/1\n{}\n/]
 
-The above request gets rejected, although the user has `read` privilege on the
-concrete index that the `current_year` alias points to. The correct permission
-would be as follows:
+To retrieve documents from `current_year`, the user must have the `read` index privilege for the alias.
 
 [source,js]
---------------------------------------------------
+----
 {
   "names" : [ "current_year" ],
   "privileges" : [ "read" ]
 }
---------------------------------------------------
-// NOTCONSOLE
-
-[discrete]
-==== Managing aliases
```

Comment:
I assume this section is removed because similar information is available on the `alias` page? Can we add a link here again to say something like "please see here for how to manage alias"?

Reply:
That's correct. I'll add the following sentence to the intro:

> For information about managing aliases, see <<alias>>.